frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/styles.scss

```diff
@@ -0,0 +1,13 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+@import 'typography';
+
+.lineage-message {
+    margin-top: -$spacer-3;
```

Comment:
Negative margins are usually a hacking thing we do from time to time.
Is there another alternative in this case?

Reply:
it seems like the list is what introduces that extra margin at the bottom so I can try removing that and see if that does it